### ISSUE TITLE
Updating to “linter-flake8@1.13.4” failed.

### ISSUE THREAD

**Issue by CarlosLopezMolina** [AtomLinter/linter-flake8#247]
Updating to “linter-flake8@1.13.4” failed.
EACCES, unlink '/home/clm/.atom/packages/linter-flake8/.npmignore'

This happens upgrading from linter-flake8 v1.9.3 in atom package manager

**Comment by Arcanemagus** [comment]
You didn't happen to install the package as `sudo apm install linter-flake8` did you? I've seen some people try that for some reason...

In any case, _something_ prevented Atom/APM from being able to delete that file when updating. If the package is currently broken, just shut down Atom, delete the folder (`rm -Rf ~/.atom/packages/linter-flake8`), and then re-install it (`apm install linter-flake8`).

**Comment by CarlosLopezMolina** [comment]
I guess that you are right. Before deleting anything, I checked that folder(~/.atom/packages/linter-flake8) and it had root as owner. The other atom package folders were owned by an unprivileged user.

As of why did I it, being an ubuntu user, I'm quite used to typing: **sudo** apt-get install  <whatever> . For sure I typed **sudo** to install linter-flake8.
I followed your recommendations and everything worked fine. Now that package folder is owned by an unprivileged user.
Thanks a lot.
Best regards.
